apps/api/src/order/order.service.ts

```diff
@@ -21,20 +21,29 @@ export class OrderService {
     @InjectModel(Order.name) private readonly orderModel: Model<Order>,
     private cartService: CartService,
     private productService: ProductsService,
-    private jwtSerivce: JwtService,
+    private jwtService: JwtService,
     private userService: UsersService,
     private authService: AuthService,
   ) {}
+  async findAllOrders(userId: any): Promise<Order[]> {
+    return await this.orderModel.find({ userId });
+  }
+
+  async findOrder(orderId: string, req: any): Promise<Order> {
+    const order = await this.orderModel.findOne({ _id: orderId });
 
-  // Objectives:
-  // Merchant Shipping Fee
-  // Product Matching - for loop userCart
-  // Product Inventory - Update Quantity
-  // Computation for Total Price
-  // Delete Items from Cart
+    if (req._id.toString() !== order.userId.toString()) {
+      throw new ForbiddenException('User is unauthorized!');
```

Comment:
change to UnauthorizedException, see reference below:
![image](https://github.com/TreshAnn/POC-EyCommerce/assets/132859942/83a0571f-380c-44fc-b1fc-154c4b376e01)